src/python/tests/core/bot/tasks/fuzz_task_test.py

```diff
@@ -730,6 +730,7 @@ def test_existing_unreproducible_testcase(self):
             job_type='job',
             fuzz_target=data_types.FuzzTarget(engine='engine', binary='binary'),
             redzone=111,
+            disable_ubsan=False,
```

Comment:
would it make sense to have a test to verify `get_ubsan_disabled_options` is working? All tests here and in #1257 are using `disable_ubsan=False`

Reply:
added.